### DISCUSSION TITLE
Welcome to Webstudio Discussions!

### DISCUSSION THREAD

**Discussion by kof** [webstudio-is/webstudio-community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GSCrawley** [comment]
Hi there! My name is Gideon; I'm currently pursuing a Master's degree in Data Science with a focus on Machine Learning. During my BSc my focus was front end development; I stumbled across Webstudio while looking for a no/low-code front end solution for a diagnostic tool I'm involved with building. I'm hoping to find some advice here
